packages/eslint-plugin/docs/rules/prefer-readonly-parameter-types.md

```diff
@@ -129,6 +129,103 @@ interface Foo {
 
 ## Options
 
+### `allow`
+
+Some complex types cannot easily be made readonly, for example the `HTMLElement` type or the `JQueryStatic` type from `@types/jquery`. This option allows you to globally disable reporting of such types.
+
+Each item must be one of:
+
+- A type defined in a file (`{from: "file", name: "Foo", source: "src/foo-file.ts"}` with `source` being optional)
```

Comment:
We should explain what it means to set `source` (i.e. is it relative or absolute?) as well as what it means to not set `source`.

Right now it's ambiguous and unclear what it means, IMO.

Reply:
Thanks, good catch, better explanation added.